sim_consistency/tasks/model_similarity.py

```diff
@@ -301,6 +300,21 @@ def compute_sim_matrix(
             corr_method=corr_method,
             max_workers=max_workers
         )
+    elif sim_method =='gromov':
+        model_similarity = GWModelSimilarity(
```

Comment:
Shall we add a warning when running this sim_method that not a sim_matrix but a dist matrix is returned?

Reply:
Yes, why not